backend/Recodo.BLL/Services/EmailService.cs

```diff
@@ -3,7 +3,7 @@
 using SendGrid.Helpers.Mail;
 using System.Threading.Tasks;
 
-namespace Thread_.NET.BLL.Services
+namespace Recodo.BLL.Services
```

Comment:
😆 👍

Reply:
Yes, i found it after merging dev, don`t know, who it was))